gtda/diagrams/features.py

```diff
@@ -32,6 +32,10 @@ class PersistenceEntropy(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
+    normalize_nb_points : bool, optional, default: ``False``
```

Comment:
I think we can find a more intuitive name here.  What about just "normalize"?  Right now, I don't think `nb_points` says much to people who don't already know what we mean.

Reply:
Ok!